reana_workflow_controller/rest.py

```diff
@@ -1080,9 +1080,11 @@ def get_workflow_status(workflow_id_or_name):  # noqa
             current_command = ''
             total_jobs = run_info.planned
             current_job_commands = _get_current_job_commands(run_info.id_)
+            current_job_names = _get_current_job_names(run_info.id_)
             try:
                 current_job_id, current_command = current_job_commands.\
                     popitem()
+                _, current_job_name = current_job_names.popitem()
```

Comment:
When you query for status just after the workflow creation there are no jobs yet, so this pop doesn't instantiate an empty `current_job_name` variable:
```console
$ reana-client start
workflow.2 has been started.
$ reana-client status
Workflow status could not be retrieved:
local variable 'current_job_name' referenced before assignment
$ reana-client status
NAME       RUN_NUMBER   CREATED               STATUS    PROGRESS   COMMAND                                                                                                                                                                                                                                                    STEP
workflow   2            2018-07-27T13:42:38   running   0/2        root -b -q '/reana/users/00000000-0000-0000-0000-000000000000/workflows/8268e9fd-4889-48ab-ac6d-af7f9e0a036d/code/gendata.C(20000,"/reana/users/00000000-0000-0000-0000-000000000000/workflows/8268e9fd-4889-48ab-ac6d-af7f9e0a036d/gendata/data.root")'   gendata
```